libcxx/include/__type_traits/add_lvalue_reference.h

```diff
@@ -18,7 +18,7 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-#if __has_builtin(__add_lvalue_reference)
+#if __has_builtin(__add_lvalue_reference) && __GNUC__ < 15
```

Comment:
```suggestion
#if __has_builtin(__add_lvalue_reference) && !defined(_LIBCPP_COMPILER_GCC)
```
We usually don't test the version since we support only one and these builtins are new anyways, so there isn't much use to check the version.

Reply:
Done.